media/js/src/simulations/simulation4/simulationFour.jsx

```diff
@@ -45,6 +45,29 @@ export const SimulationFour = () => {
         </ul>
     </div>;
 
+    const mkRealDataBtn = (progress, stage, handleProgress) => {
```

Comment:
You should be able to drop `handleProgress` from the argurments.

Reply:
Real data button will always need to trigger the progress

Reply:
Right but I don't think the function itself needs to be passed in as an argument in order to be used

Reply:
Yeah I guess if it stays on this page. I'll make the change